config/configuration.toml.in

```diff
@@ -20,6 +20,9 @@ default_vcpus = -1
 #default_memory = @DEFMEMSZ@
 disable_block_device_use = @DEFDISABLEBLOCK@
 
+# Enable pre allocation of VM RAM, default false
```

Comment:
We probably want to explain why we'd want to set that one to true and what would be the consequences as far as memory footprint would be (i.e. bigger).